worker-languagedetection-container/testcases/automated/CAF_4002/cebuano_ganda.yaml.testcase

```diff
@@ -53,35 +53,35 @@ expectedOutputData:
       DetectedLanguage1_ConfidencePercentage:
         action: "replace"
         values:
-        - data: "55"
+        - data: "48"
```

Comment:
Remove this whole testcase as not supporting Ganda as a language

Reply:
Done